csharp/Platform.Collections/IDictionaryExtensions.cs

```diff
@@ -4,17 +4,57 @@
 
 #pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
 
-namespace Platform.Collections
+namespace Platform.Collections 
 {
+    /// <summary>
+    /// <para>Represents a set of extension methods <see cref="IDictionary{TKey, TValue}"/> array.</para>
+    /// <para>Представляет набор методов расширения <see cref="IDictionary{TKey, TValue}"/>.</para> 
+    /// </summary>  
     public static class IDictionaryExtensions
     {
+        /// <summary>    
+        /// <para>We get the default value.</para> 
+        /// <para>Получаем дефолтное значение.</para> 
+        /// </summary>  
+        /// <typeparam name="dictionary"> 
+        /// <para>Dictionary that is a key pair:meaning.</para> 
```

Comment:
`An <see cref="IDictionary<TKey, TValue>"/> from which to find the value associated with the <paramref name="key"/>.`